next-client/src/components/navbar/components/LoginButton.tsx

```diff
@@ -72,7 +72,7 @@ export default function LoginButton({
   if (loading) {
     return (
       <div>
-        <p>Loading...</p>
+        <Button disabled={true}>...</Button>
```

Comment:
I may have gotten a little carried away with UI fixes, can revert this if people don't like it